docs-src/stargate-develop/modules/develop/examples/result/rest_curl_get_all.result

```diff
@@ -13,15 +13,15 @@
     }, 
     {
       "firstname": "Janesha",
-      "evaluations": 
+      "evaluations": [
         {
-          "2019" : "okay",
-          "2020" : "good"
+          "key": 2019,
+          "value": "okay"
         },
-      "top_three_tv_shows": [
```

Comment:
Whops. Need to keep this, was accidentally deleted.

Reply:
Thanks - fixing..

Reply:
See most recent commit - thanks.